windows/libcommon/include/s2e/ModuleMap.h

```diff
@@ -0,0 +1,93 @@
+/// S2E Selective Symbolic Execution Platform
```

Comment:
Does it make more sense to have this file in `guest/common/include`? That way you don't have to duplicate the structs [here](https://github.com/S2E/libs2eplugins/blob/55ae7d338a416a2019db0359e79b60c4c4e4efd2/src/s2e/Plugins/OSMonitors/Support/ModuleMap.h)

Reply:
Wait no, maybe this isn't possible. I didn't realize `guest/windows/libcommon` had its own `s2e.h`

Reply:
This is windows stuff. I agree that it would be nice to have a common cross-platform file, but that's for another issue.

Reply:
Hmmm ok. Can you at least add a note here to say that if this file is updated, the duplicate code in libs2eplugins must also be updated.